Makefile

```diff
@@ -272,3 +268,16 @@ clean-images:
 test-docker-clean: ## Clean up Docker environment used in test-docker target
 	-docker-compose down --rmi all
 
+
+##@ Tooling
+
+.PHONY: update-satellite-cfg-lock
+update-satellite-cfg-lock: ## Update the satellite config lock file
+	@docker run -ti --rm \
```

Comment:
This docker seems a large overkill. Cannot we add something like `satellite print-config >> ./scripts/satellite.yaml.lock`?

Reply:
We could but it's an improvement to something that it's already in master and it doesn't do what's expected, moreover this must be run by a developer machine and it shouldn't be often used so the delay that there is in running in docker that it's less than a minute in my machine shouldn't stop to fix the problem with this approach.

Making the improvement that you mentioned would be desired, however I see that it's more work than the one planned for the current ticket, but I may be wrong with it. If the amount of work to add such sub-command to the satellite is no brainer I would like some guidance with it, because I don't have enough knowledge on the code base to see it in such way.